common/djangoapps/student/models.py

```diff
@@ -900,6 +900,20 @@ def is_allowable_password_reuse(cls, user, new_password):
 
         return True
 
+    @classmethod
+    def retire_user(cls, user_id):
+        """
+        Updates the password in all rows corresponding to a user
+        to an empty string as part of removing PII for user retirement.
+        """
+        changed_password = False
+        with transaction.atomic():
+            for row, _ in cls.objects.filter(user_id=user_id):
+                changed_password = True
+                row.password = ""
+
```

Comment:
Do we need the save method?

Reply:
I had it, but the test passes without it. Unless the test is wrong, I don't think we do need it.